shell/src/main/java/org/apache/accumulo/shell/commands/HiddenCommand.java

```diff
@@ -42,9 +43,9 @@ public String description() {
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws Exception {
     if (rand.nextInt(10) == 0) {
-      shellState.getReader().beep();
-      shellState.getReader().println();
-      shellState.getReader()
+      shellState.getTerminal().puts(InfoCmp.Capability.bell);
```

Comment:
Have you tested the bell to make sure it actually rings? (might depend on your local development environment).

Reply:
I have not but I will try on my end. This was how the devs on the mailing list said to do it. I never heard the bell on the old shell so I am not sure if I will hear this one.

Reply:
The bell should be the same one you hear in Linux if you got backspace repeatedly on an empty line. If you can't hear it by trying that, then it's probably configurable in your system sound settings to enable it.

Reply:
Confirmed that the bell is there with these changes. Had to do a bit of settings adjustments in my terminal to make it audible.